.github/workflows/milestone.yml

```diff
@@ -12,9 +12,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Get milestone from pom.xml
-        run: |
-          .github/bin/retry ./mvnw -v
```

Comment:
Retry here was to retry maven download in case of network issues.

Reply:
I don't believe this was the case here. It downloads only maven jar, without project dependencies.
Have you got an experience that maven jar download is not stable?

Reply:
> Yes. That was a case in the past.

Interesting. We have much more artifacts to download `from repo1.maven.org/maven2` and I could bet maven zip itself is not a bottle neck.

> Also "maven jar" consists of plenty of jars.

But they are compressed to one file:
```
distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.8.4/apache-maven-3.8.4-bin.zip
```
https://github.com/apache/maven-wrapper/blob/master/maven-wrapper/src/main/java/org/apache/maven/wrapper/Installer.java

I'd leave these line removed If only you don't insist to revert a change.

Reply:
Assuming the network to be flaky is a better assumption that the other. Retries don't hurt, so we should keep it here IMO.

Reply:
2:1